doc/timeseries.md

```diff
@@ -7,9 +7,36 @@ The purpose of this extension is to provide a cohesive user experience around th
 [![Static Badge](https://img.shields.io/badge/%40tembo-community?logo=slack&label=slack)](https://join.slack.com/t/tembocommunity/shared_invite/zt-277pu7chi-NHtvHWvLhHwyK0Y5Y6vTPw)
 [![OSSRank](https://shields.io/endpoint?url=https://ossrank.com/shield/4022)](https://ossrank.com/p/4022)
 
+## Installation
+
+### Running with docker
+
+Start a Docker container running Postgres with `pg_timeseries` pre-installed.
+
+```bash
+docker run -d --name pg-timeseries -p 5432:5432 -e POSTGRES_PASSWORD=postgres quay.io/tembo/timeseries-pg:latest
+```
+
+Then connect to the database and enable the extension:
+
+```bash
+psql postgres://postgres:postgres@localhost:5432/postgres
+```
+
+```sql
+CREATE EXTENSION timeseries CASCADE;
```

Comment:
In the PostgreSQL docker image, I believe there is a folder on disk you can put `sh` and `sql` files in and they'll be run on first startup… maybe `CREATE EXTENSION` could be added there so the user doesn't have to run it?

This is documented [here](https://github.com/docker-library/docs/blob/master/postgres/README.md#initialization-scripts). You'd just need to make an `sql` file with `CREATE EXTENSION` in it, then `COPY` it to that folder during the build.